doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -82,12 +82,12 @@ IAM role-based operator deployment
 Before you can deploy your operator using an IAM role, associate an OpenID Connect (OIDC) provider with your role to
 authenticate with the IAM service.
 
-Associate an OpenID Connect Provider to Your Instance
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create an OpenID Connect Provider for Your Cluster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Create an OIDC identity provider for your cluster. If your
-cluster is managed by EKS, then your cluster will already have an OIDC
-attached to it. 
+Create an OIDC identity provider for your cluster.
```

Comment:
Why are we repeating the header here? Also is it `OpenID Connect Provider` or `OIDC identity provider`?